src/components/chat/chat-thread.tsx

```diff
@@ -63,9 +65,15 @@ export const ChatThread: React.FC<IProps> = ({ activeNavTab, user, chatThreads,
               "chat-thread-focused": shouldBeFocused,
             })}
             data-testid="chat-thread">
-            <div className="chat-thread-header" onClick={() => handleThreadClick(key)}> 
+            <div className={classNames(`chat-thread-header ${activeNavTab}`,
+              { "selected": shouldBeFocused })}
+              data-testid="chat-thread-header"
+              onClick={() => handleThreadClick(key)}
+            > 
               <div className="chat-thread-tile-info">
-                {Icon && <Icon/>}
+               {Icon && (
+                <div data-testid="chat-thread-tile-type"><Icon/></div>
```

Comment:
Could the `data-testid` be applied directly to the `Icon`?

Reply:
Yes! Looks like it can. Good idea. Thanks!